CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 ### Fixed
 
 - Fix duplication of Materials in Transit asset value in Long-Term Materials Receivable
+- Fix materials in the PROD and PRODQ buffers not being clickable
```

Comment:
```suggestion
- `prun-bugs`: Fix material icons in the PROD and PRODQ buffers not being clickable
```

Also, the changelog sections are following this structure to improve readability:
1. XIT-related notes
2. Feature-related notes
3. Other changes

(1) and (2) are sorted alphabetically. While you are at it, fix the structure of this changelog section please.

<img width="601" alt="image" src="https://github.com/user-attachments/assets/ca182107-53c1-4b09-b29b-fdb7cbd922bb" />